handbook/marketing/README.md

```diff
@@ -294,7 +294,7 @@ We use Figma for most of our design work. This includes the Fleet product, our w
 
 ##### Which file should I use?
 
-**Fleet product** All product design work is done in the [Fleet EE (scratchpad)](https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE-(dev-ready%2C-scratchpad)?node-id=9209%3A302838) Figma doc. Check out the [README](https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE-(dev-ready%2C-scratchpad)?node-id=2750%3A67203) for how to use this doc.
+**Fleet product.** All product design work is done in the [Fleet product](https://www.figma.com/files/project/17318630/%F0%9F%94%9C%F0%9F%93%A6-Fleet-EE%C2%AE-(product)?fuid=1234929285759903870) Figma project. Check out the [Wireframing](https://fleetdm.com/handbook/product#wireframing) for more details.
```

Comment:
I've noticed several links styled as `[📖<page>#<section>]` when linking between handbook pages, think it makes sense to do that here? (Kinda nice how it shows you're navigating to a different top-level section.) 
Also, spotted an extra "the":
```suggestion
**Fleet product.** All product design work is done in the [Fleet product](https://www.figma.com/files/project/17318630/%F0%9F%94%9C%F0%9F%93%A6-Fleet-EE%C2%AE-(product)?fuid=1234929285759903870) Figma project. Check out [📖Product#wireframing](https://fleetdm.com/handbook/product#wireframing) for more details.
```

Reply:
@rachaelshaw Just updated. I noticed that we use these links to different handbook pages in different ways. Sometimes it is "📖Page/Section name" or "📖Page#Section name" and sometimes "📖Page#section-name". I've used the first approach since it looks more readable to me.